aws_lambda_builders/workflows/python_pip/utils.py

```diff
@@ -16,6 +16,19 @@ class OSUtils(object):
     def environ(self):
         return os.environ
 
+    def original_environ(self):
+        # https://pyinstaller.readthedocs.io/en/stable/runtime-information.html#ld-library-path-libpath-considerations
+        env = dict(os.environ)
+        lp_key = "LD_LIBRARY_PATH"
+        original_lp = env.get(lp_key + "_ORIG")
```

Comment:
is this just specific to some systems where `LD_LIBRARY_PATH_ORIG` is populated? is there any way we can make this very specific to pyinstaller? does pyinstaller re-introduce any other env vars?

Reply:
LD_LIBRARY_PATH_ORIG is only populated if LD_LIBRARY_PATH is set. There are no other env vars that are generated by PyInstaller. I will add a check to run this only in Linux.